### DISCUSSION TITLE
/chat Rooms GUI

### DISCUSSION THREAD

**Discussion by SenPr** [Zrips/CMI#7024] [Feature Requests]
I think the /chat command is very cool, but not used a lot due to how complicated and confusing the commands can be for most players. At least from what I have observed from my players on my server.

Having a GUI for this feature would greatly improve this. Players can see existing public rooms, they can do functions such as opening a new room, sending invites, making private rooms, and more through the use of the GUI rather than typing /help or /chat everytime they need to do something.

**Comment by mrfloris** [comment]
I really like this idea, i wonder how convinience of going through a GUI menu versus a command is - or what a logical order is of 'chat rooms you are in, are invited to, can join, request invite maybe?, and finally listed as private'

maybe some sort of order button to cycle through them is needed, but by default show the ones you're in (with glow effect) and are invited to (no glow effect)
